lib/bloc_record/selection.rb

```diff
@@ -125,6 +127,11 @@ def where(*args)
     rows_to_array(rows)
   end
 
+  def not(hash)
+    str_condition = hash.map {|k,v| "#{k} != '#{v}'"}.join(" AND ")
```

Comment:
- [ ] The only thing I'd be wary of is automatically including the "AND" here. You may want that to be optional.